src/utils/public-ip.ts

```diff
@@ -28,8 +28,10 @@ import { hostname } from 'os'
 import getIp from './ip'
 import { sendPing } from './ping'
 import { sendHttpRequest } from './http'
+import { getContext } from '../context'
+import { isSymonModeFrom } from '../components/config'
 
-export let publicIpAddress = ''
+export let publicIpAddress = '0.0.0.0'
```

Comment:
i think it's better to use something like `x.x.x.x` to tell that it's not an actual IP Address

Reply:
good point